examples/data-block-examples/sentinel2-l2-a-geotiff-example.ipynb

```diff
@@ -0,0 +1,302 @@
+   "execution_count": 2,
+   "id": "static-riding",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import SDK\n",
+    "import up42"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "scientific-haiti",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-03-19 16:45:59,477 - Authentication with UP42 successful!\n",
+      "2021-03-19 16:46:00,349 - Initialized Project(name: S2l2a-Analytics-data-example, project_id: 1f2bad99-daad-4a60-9192-5169263caff6, description: , createdAt: 2021-03-18T14:39:08.255254Z)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Authenticate user and initialise project\n",
+    "up42.authenticate(project_id=\"5678\", \n",
+    "                  project_api_key=\"uvwx\")\n",
+    "project = up42.initialize_project()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "regulated-prospect",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define the aoi\n",
+    "aoi = {\n",
+    "  \"type\": \"FeatureCollection\",\n",
+    "  \"features\": [\n",
+    "    {\n",
+    "      \"type\": \"Feature\",\n",
+    "      \"properties\": {},\n",
+    "      \"geometry\": {\n",
+    "        \"type\": \"Polygon\",\n",
+    "        \"coordinates\": [\n",
+    "          [\n",
+    "              [\n",
+    "                14.610298,\n",
+    "                -24.516841\n",
+    "              ],\n",
+    "              [\n",
+    "                14.622478,\n",
+    "                -24.516841\n",
+    "              ],\n",
+    "              [\n",
+    "                14.622478,\n",
+    "                -24.50833\n",
+    "              ],\n",
+    "              [\n",
+    "                14.610298,\n",
+    "                -24.50833\n",
+    "              ],\n",
+    "              [\n",
+    "                14.610298,\n",
+    "                -24.516841\n",
+    "              ]\n",
+    "          ]\n",
+    "        ]\n",
+    "      }\n",
+    "    }\n",
+    "  ]\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "stupid-pathology",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-03-19 16:46:11,248 - Getting existing workflows in project ...\n",
+      "2021-03-19 16:46:11,718 - Got 1 workflows for project 1f2bad99-daad-4a60-9192-5169263caff6.\n",
+      "100%|██████████| 1/1 [00:00<00:00,  1.50it/s]\n",
+      "2021-03-19 16:46:12,396 - Using existing workflow: Sentinel2-l2-analytics-example - 00e94c32-7cfe-40b8-85b6-cc2ffa980c75\n",
+      "2021-03-19 16:46:16,343 - Added tasks to workflow: [{'name': 'esa-s2-l2a-gtiff:1', 'parentName': None, 'blockId': '4471e5ef-90f1-4bf0-9243-66bc9d8b4c99'}, {'name': 'sharpening:1', 'parentName': 'esa-s2-l2a-gtiff:1', 'blockId': 'e374ea64-dc3b-4500-bb4b-974260fb203e'}]\n"
```

Comment:
This is unfortunately not the right block. Your workflow is called analytic example, hence you should use "esa-s2-l2a-gtiff-analytic" block. It's a bit confusing as there are several blocks on the platform right now with similar name

Reply:
Thank you, I will begin to correct this.